crates/nu-command/src/system/run_external.rs

```diff
@@ -93,6 +110,7 @@ pub fn create_external_command(
     call: &Call,
     redirect_stdout: bool,
     redirect_stderr: bool,
+    out_file: Option<Spanned<String>>,
```

Comment:
This doesn't feel right to me. `create_external_command` shouldn't need to know that its output will be going to a file.